src/test/kotlin/ar/com/dgarcia/javaspec/testSpecs/KotlinLocalLetSpecTest.kt

```diff
@@ -1,26 +1,24 @@
 package ar.com.dgarcia.javaspec.testSpecs
 
-import ar.com.dgarcia.javaspec.api.JavaSpec
 import ar.com.dgarcia.javaspec.api.JavaSpecRunner
-import ar.com.dgarcia.javaspec.api.contexts.TestContext
-import ar.com.dgarcia.javaspec.api.variable.Let
-import ar.com.dgarcia.javaspec.api.variable.set
+import ar.com.dgarcia.kotlinspec.api.KotlinSpec
+import ar.com.dgarcia.kotlinspec.api.variable.Let
 import org.assertj.core.api.Assertions.assertThat
 import org.junit.runner.RunWith
 
 /**
- * This type tests usage of local lets in Kotlin
+ * This type tests usage of lets in Kotlin
  * Created by nrainhart on 11/05/19.
  */
 @RunWith(JavaSpecRunner::class)
-class KotlinLocalLetSpecTest : JavaSpec<TestContext>() {
+class KotlinLocalLetSpecTest : KotlinSpec() {
```

Comment:
Con esta definición estamos diciendo que en kotlin NO tenemos tests que usen interfaces...
Tal vez sea la forma que queremos y simplemente me estoy aferrando a lo conocido, pero tener algo combinado sería muy complicado de implementar?
No se me ocurre ahora, pero me imagino que tal vez tengas cosas que las queres sacar del "context" y otras como variables locales?

A lo que voy es que la limitación de no poder combinarlos sería artificial. Entiendo que tal vez esa limitación simplifica el uso y tal vez sea mejor. Qué opinas?

Reply:
Sí, fue una decisión consciente para simplificar el uso. Para mí, incluir siempre el TestContext agrega una carga cognitiva sobre el usuario desde el momento 0. O sea, no sólo tenés que entender qué es KotlinSpec, sino también qué son los contexts (aunque no los vayas a usar directamente).

Otra razón por la que no los incluí es, porque usar TestContexts con Suppliers en Kotlin la verdad que es bastante verboso. Igual ese no es un argumento muy fuerte, porque también se podría implementar un TestContext propio de Kotlin. Respecto a la complejidad de implementarlo... no es tan trivial como el let, porque tenés que meterte un poco más adentro de JavaSpec, pero tampoco debería ser una locura.

De todos modos, me parecía bien arrancar así, y después ver si tiene sentido agregar lo otro.

Reply:
Dale. Vamos con esa. De ultima volver para atrás no sería dificil teniendo que soportar los dos mundos (java y kotlin)

Reply:
Y tengo que venir hasta acá para enterarme de esta decisión señores! así no va!!! 
Justo quería usarlos combinados.. venía con context y se me ocurrió probar locaLet(para probar un error de java/kotlin `Variable [numberOfConfirmedGuests] has mismatching type definitions in get [double] and let [java.lang.Double] operations` ), pero bueno.. cambio de idea entonces..